test/tests.json

```diff
@@ -1416,8 +1416,7 @@
     {
       "data": "`null`",
       "expression": "round(1.24349E2, 2)",
-      "error": "syntax",
-      "was": 124.35
+      "expected": 124.35
```

Comment:
@JohnBrinkman any reason why this was considered a syntax error earlier. This seems intentional.

Reply:
Hmm.  Not sure of the history.  But before your changes that expression would give us:
`Error: ArgumentError: round() takes 2 arguments but received 3`
So it was an error because it didn't parse the exponent correctly.

Reply:
okay. I will merge the PR then.